packages/camera/camera/example/test/flutter_test_config.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:leak_tracker_flutter_testing/leak_tracker_flutter_testing.dart';
+
+Future<void> testExecutable(FutureOr<void> Function() testMain) async {
```

Comment:
I'm not sure how this is ran. Does this run automatically with `flutter test`? Could you add a comment that explains how this is used or provide a doc link?

Reply:
Yes this is run automatically with `flutter test`. Here is the documentation https://api.flutter.dev/flutter/flutter_test/flutter_test-library.html

I don't mind adding a comment or something if you feel it is needed. Could you tell me what kind of comment you are expecting?

Reply:
Ah, ok. Sorry I was unfamiliar with this in `flutter_test`. As long as there is public documentation for this then it doesn't need a comment.